telegram/ext/filters.py

```diff
@@ -1050,11 +1051,34 @@ def filter(self, message):
                 if self.usernames:
                     return (message.from_user.username
                             and message.from_user.username in self.usernames)
+                return False
+            else:
+                return self.allow_empty
+
+    class via_bot(user):
+        """Filters messages to allow only those which are from specified via_bot ID(s) and/or
+        username(s).
+
+        Warning:
```

Comment:
Please copy the code from `Filters.user` and add explicit doc-strings. The method names of `Filters.user` are not really appropriate here, e.g. `remove_user_ids` should be renamed to `remove_bot_ids`  here. Same applies for the variable names.